service/src/main/java/com/ge/predix/acs/privilege/management/PrivilegeManagementServiceImpl.java

```diff
@@ -290,6 +297,23 @@ public BaseSubject getBySubjectIdentifier(final String subjectIdentifier) {
         return subject;
     }
 
+    @Transactional(readOnly = true)
```

Comment:
Line 289:  getBySubjectIdentifier  - this method needs to use a different implementation on the graph repository, such that it *only* returns the attributes explicitly set on the subject. Currently this will return all inherited attributes as well.